fsspec/implementations/local.py

```diff
@@ -102,7 +102,7 @@ def put_file(self, path1, path2, **kwargs):
     def mv_file(self, path1, path2, **kwargs):
         path1 = self._strip_protocol(path1).rstrip("/")
         path2 = self._strip_protocol(path2).rstrip("/")
-        os.rename(path1, path2)
```

Comment:
`mv_file` method does not inherit the interface of `AbstractFileSystem` class. It looks a little strange to me.

Reply:
In the base, we have cp which calls cp_file and rm which calls rm_file. The base mv uses these two, but here (and some other cases) we have a single call that will do it.